### ISSUE TITLE
Update the Marketplace preview icon?

### ISSUE THREAD

**Issue by Gama11** [naumovs/vscode-ext-color-highlight#8]
Hey, great extension! :+1: 

I noticed that the highlighting looks quite different (a lot better in my opinion) than in the preview image used on the marketplace. Looks like this changed in #3, and the image wasn't updated?

Anyway, thought I'd mention it since you might get more installs with a newer image. ;)

**Comment by lenivaya10001** [comment]
yes, gama11 right... the logo with color dots after color syntax will x3 increase magic power of extension logo. The color dots is cool!

**Comment by manskou** [comment]
I almost didn't install this because I thought it did what the icon was showing. I came here to see if there was an option to make it look more like Atom's color highlighting and then I realized that that's what it does.

The icon needs to be updated for sure!

**Comment by baktybek0v** [comment]
why is the icon still not updated?
